var/spack/repos/builtin/packages/camp/package.py

```diff
@@ -3,9 +3,19 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import glob
+
 from spack.package import *
 
 
+def hip_repair_options(options, spec):
+    # there is only one dir like this, but the version component is unknown
+    options.append(
+        "-DHIP_CLANG_INCLUDE_PATH="
+        + glob.glob("{}/lib/clang/*/include".format(spec["llvm-amdgpu"].prefix))[0]
+    )
```

Comment:
I think this needs to be changed slightly to resolve my issue:

```python
path = glob.glob("{}/lib/clang/*/include".format(spec["llvm-amdgpu"].prefix))[0]
options.append(cmake_cache_path("HIP_CLANG_INCLUDE_PATH", "{0}".format(path))

Reply:
I have verified that with a camp already built, this patch fixes my problem. I will probably make a PR to add this change, but want to make sure I'm not breaking things elsewhere/in camp

Reply:
I see. Can/Will that fix also be applied to RAJA?

Reply:
I see there is a RAJA PR linked to this as well, so I assume so. Thanks for the help!